MNE/rtProcessing/ttt.h

```diff
@@ -0,0 +1,11 @@
+#ifndef TTT_H
+#define TTT_H
+
+
+class ttt
+{
+public:
+    ttt();
+};
+
+#endif // TTT_H
```

Comment:
What kind of class is this? is it getting used somewhere? If not, please delete and recommit. If you need new helper class in the the rtProcessing library, please put them into MNE/rtProcessing/helpers  folder (non existent yet, you would need to create it). cheers, lorenz